src/it/sample-plugin/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Why don't you just build several reference plugins with the tweaked POM? (e.g. particular releases, which are known to be working fine). I think it is fine to have some bleeding edge builds for this repo